tests/unit/components/UI/Text/InputText.spec.js

```diff
@@ -0,0 +1,16 @@
+import { mount } from "@vue/test-utils";
+import InputText from "@/components/UI/Text/InputText";
+import Tooltip from "primevue/tooltip";
+
+describe("InputText.vue", () => {
+  it("should render with default values", () => {
+    const wrapper = mount(InputText, {
+      global: {
+        directives: {
+          tooltip: Tooltip,
```

Comment:
Do you think we should have the global directives also mentioned like global components? In one file `jest.init.js` instead of loading with every mount?

Reply:
Agreed